src/screens/EthToSub.tsx

```diff
@@ -200,6 +206,35 @@ const EthToSub = ({ className, ethProvider } : Props) => {
 	);
 };
 
+function parseAmount(val: string|null) {
+	if (!val) {
+		return val;
+	}
+
+	return val.replace(/k/gi, '000').replace(/m/gi, '000000');
```

Comment:
I'm worried that undocumented/uncommented features would creep the code. Is it that often that you need to send large amounts? A little note somewhere would be great, or maybe just showing this off in the placeholder?

Reply:
Added in the placeholder and added docs to the function itself.